.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    "configurations": [
+        {
+            "name": "Python Debugger: Remote Attach",
+            "type": "debugpy",
+            "request": "attach",
+            "connect": {
+                "host": "localhost",
+                "port": 5678
+            },
+            "pathMappings": [
+                {
+                    "localRoot": "${workspaceFolder}/src",
+                    "remoteRoot": "/app"
+                },
+                {
+                    "localRoot": "~/.pyenv/versions/3.13.1/envs/venv3.13.1/lib/python3.13/site-packages/",
+                    "remoteRoot": "/usr/local/lib/python3.13/site-packages/"
```

Comment:
zijn dit soort settings niet heel device specifiek? Mag je prima in je eigen map houden, maar misschien met een `.gitignore`? (en eventueel een `launch.json.example` file als dat handig is)